[BBIM docs] change color of ui breadcrumb on dark background

dirkolbrich opened the pull request:
Make the breadcrumb text better readable on dark background.
ToDo: find a way to change the color of the SVG symbols as well.

Before:
![BlenderBIM-docs_breadcrumb_before](https://github.com/IfcOpenShell/IfcOpenShell/assets/3765119/11337537-c378-4a5c-8509-623ba4034ece)

After commit:
![BlenderBIM-docs_breadcrumb_after](https://github.com/IfcOpenShell/IfcOpenShell/assets/3765119/21f5f2a0-ab3e-4f0a-9ada-eaa0bedb0f1f)

Moult commented:
There is actually a dark mode vs light mode override in the sphinx conf, can you check that out? Sorry afk right now.

Moult commented:
Yep, check src/blenderbim/docs/conf and redefine the CSS variable for dark mode there.

dirkolbrich commented:
There are no overrides in the `conf.py` file for the breadcrumb style (called Admonitions in the furo theme).

The current `custom.css` uses the same color for the border, text and icons (here hardcoded into the SVG file).

https://github.com/IfcOpenShell/IfcOpenShell/blob/a5d123fe833da1e37f77c3e557811ff8bf2d946d/src/blenderbim/docs/_static/custom.css#L61-L67

There is no separate css class defined for admonitions text color and border within the Furo theme. If I  override the `color-admonition-title` class within the `conference.py` it also changes the color border of the breadcrumb. That's not what I wanted to achieve.

Moult commented:
Yep, I'm saying rather than change it globally, to use a CSS variable to define the separate light and dark colours. That way light can stay as the purple colour. The conf file can then define the CSS variable.

dirkolbrich commented:
Please reopen this PR, so Github adds the follow-up commits and makes them visible in this PR.

I think you don't understand. I just wanted to change the text color, not the border color as well. 

If the `volor-admonition-title` attribute is overridden in `conf.py`, it also changes the border color. There needs to be a distinction between text and border within `custom.css`. I added a new CSS class to `custom.css` just for the text and added color overrides to `conf.py`

Look at the other default admonitions of the Furo theme, they also use a different color class for the text and the border.

Moult commented:
Cheers, reopened PR. I understand what you're saying. What I'm suggesting is to define a _new_ or different CSS variable, and then define it in the conf.py for both light and dark.

dirkolbrich commented:
> What I'm suggesting is to define a new or different CSS variable, and then define it in the conf.py for both light and dark.

Did that with the additional commit.

Moult commented:
Beautiful! Thank you so much! Sorry for the hasty close!
